_posts/2016-05-12-beer-card-pet-projects-tgif-83.md

```diff
@@ -0,0 +1,130 @@
+
+---
+
+title: 'BEER CARD PET PROJECTS TGIF (83)'
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it’s time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usual, they’re curated for you by [Cecily](https://twitter.com/skeskali). Enjoy!
+
+
+
+
+## Open Source
+
+[16 Opensource EmberJS Projects to Learn From](https://www.icicletech.com/blog/16-opensource-emberjs-projects-to-learn-from)
+
+Hoodie’s Gregor Martynus was a guest on this week’s Toolsday podcast on [Offline First](http://toolsday.io/episodes/offline-web.html)
+
+[Knox - a service for storage and rotation of keys, passwords, and secrets used by services](https://github.com/pinterest/knox)
+
+[10 Years and over 520 Episodes of Podcasting—Tech is a Marathon, Not a Sprint](https://medium.com/@shanselman/10-years-and-over-520-episodes-of-podcasting-tech-is-a-marathon-not-a-sprint-46d75899a083#.gkw36opw8)
+
+>The term “open source” has become so broad as to become meaningless. I would guess at least half of my conversations with people involve the question “Wait…so what IS open source, actually?”, or realizing about 10 minutes in that we’re talking about completely different things.
+
+<cite>Nadia Eghbal, [I Hate the Term Open Source](https://medium.com/@nayafia/i-hate-the-term-open-source-a65fd481a95#.c0n7q1976)</cite>
+
+[A New Way to Fund Open Source Projects](https://medium.com/open-collective/a-new-way-to-fund-open-source-projects-91a51b1b7aac#.m8s4e0bzy)
+
+
+## Design
+
+>Budweiser is going to potentially ingenious, potentially absurd branding extremes. The company has kept the same can you already know, but when you look closely, you’ll realize that it has swapped out its own name, "Budweiser," for "America." That’s right, Budweiser has renamed its beer America for the summer. "We thought nothing was more iconic than Budweiser and nothing was more iconic than America," says Tosh Hall, creative director at the can’s branding firm JKR.
+
+<cite>Fast Company, [Budweiser Renames Its Beer "America"](http://www.fastcodesign.com/3059681/budweiser-renames-its-beer-america)</cite>
+
+[Why Designers Write on the Walls (And Why You Should Too)](https://medium.com/@george_aye/always-show-your-work-why-designers-write-on-the-walls-and-why-you-should-too-6725237b9765?mc_cid=c3898e3176&mc_eid=549abb3da2#.6ie4rkobz)
+
+[Resource Cards](https://resourcecards.com/?mc_cid=c3898e3176&mc_eid=549abb3da2) is a list of free resources for creatives
+
+
+## The Web and Development
+
+[Locking In Webkit](https://webkit.org/blog/6161/locking-in-webkit/)
+
+
+[Tradeoffs In Coordination Among Teams](http://blog.jessitron.com/2016/05/tradeoffs-in-coordination-among-teams.html)
+
+
+[Here’s a Slack slash command to let your teams know when a task is done.
+](https://hoodie-community.slack.com/apps/A0PB7H5AT--done)
```

Comment:
github isn't showing that this has been formed into a proper URL, is someone able to check how it renders?